recipe/meta.yaml

```diff
@@ -1,24 +1,23 @@
-{% set version = "0.1.7" %}
+{% set version = "0.1.8" %}
 
 package:
   name: erddapy
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/e/erddapy/erddapy-{{ version }}.tar.gz
-  sha256: c962cbe1adb011df7443cde486aa8e7c8ea7d83c1ad3e54c9afb9b03a9445a16
+  sha256: 97a45006a29d78f38c97b64127529c9802109b2b0a49448f61e23cbbbc5b5a89
 
 build:
   number: 0
-  noarch: python
   script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   build:
-    - python
+    - python >=3.6
```

Comment:
Removing `noarch: python` and adding python constraints is not supported. 2 Options,
1. Add noarch: python
2. Remove python constraint here and adding a skip statement.

Reply:
Good catch. Don't worry about this though. Only my old scripts running amok.
(I should be using rever for this :smile:)